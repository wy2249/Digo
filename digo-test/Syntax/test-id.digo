func digo_main() void {
  _a := 1
  println("%d", _a)
}
