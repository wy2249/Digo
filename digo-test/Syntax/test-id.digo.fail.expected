Fatal error: exception Failure("lexing: empty token")
